pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -37,6 +42,10 @@ const (
 	fetchAllPlugins = 1
 )
 
+var (
+	routeAPIFound = false
```

Comment:
Is this needed ?

Reply:
it will be used to read if the routeAPIExists. In the case of running this on Kubernetes, Operator might break.